manifests/mod/security.pp

```diff
@@ -24,6 +24,7 @@
   $secrequestbodylimit         = '13107200',
   $secrequestbodynofileslimit  = '131072',
   $secrequestbodyinmemorylimit = '131072',
+  $deploy_security_crs         = true,
```

Comment:
Could you please name this `manage_security_crs` to be more similar to other parameters in the module?